src/scss/components/file/mixins/hxFileControl.scss

```diff
@@ -0,0 +1,66 @@
+@import "vars";
+
+@mixin hxFileControl(
+  $pseudo-state: null,
+  $btnState: null,
+  $disabledState: null
+) {
+  @if $pseudo-state == focus {
+    box-shadow: $focus-glow;
+  } @else if $pseudo-state == active {
+    ~ label > hx-file-input {
+      // TODO refactor button mixins
+      @if $btnState == btn {
+        &.hxBtn {
+          background-color: $cyan-700;
+          border-color: transparent;
+          color: $gray-0;
+        }
+      } @else if $btnState == hxPrimary {
+        // TODO refactor button mixins
+        &.hxBtn.hxPrimary {
+          background-color: $cyan-700;
+          border-color: transparent;
+          color: $gray-0;
+        }
+      } @else if $btnState == hxTertiary {
+        // TODO refactor button mixins
+        &.hxBtn.hxTertiary {
+          background-color: transparent;
+          border-color: transparent;
+          color: $cyan-700;
+        }
+      }
+    }
+  } @else if $pseudo-state == disabled {
+    ~ label {
+      color: $gray-700;
+
+      > hx-file-input {
+        cursor: not-allowed;
+        @if $disabledState = normal {
+          // TODO refactor button mixins
+          &.hxBtn {
+            background-color: transparent;
+            border-color: $gray-400;
+            color: $gray-400;
+          }
+        } @else if $disabledState = hxprimary {
+          // TODO refactor button mixins
+          &.hxBtn.hxPrimary {
+            background-color: $gray-400;
+            border-color: transparent;
+            color: $gray-700;
+          }
+        } @else if $disabledState == hxtertiary {
+          // TODO refactor button mixins
+          &.hxBtn.hxTertiary {
+            background-color: transparent;
+            border-color: transparent;
+            color: $gray-400;
+          }
+        }
+      }
+    }
```

Comment:
There's no point to define these styles, because they can all be applied to spec markup via the `[disabled]` HTML attribute without the need of additional CSS. Additionally, "disabled" would map to `$state` and not `$pseudo-state`.

Knowing this, please leave these styles as they are currently defined in [`src/scss/components/file/_hx-file-control.scss`](https://github.com/HelixDesignSystem/helix-ui/blob/master/src/scss/components/file/_hx-file-control.scss#L59-L86) and remove this entire block of styles.